CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 ## Language changes
 
+* `Bit` properties (e.g. `property p = True`) will be checked with `:exhaust`,
+  unless their docstrings contain code blocks understood by `:check-docstrings`.
+  ([#1842](https://github.com/GaloisInc/cryptol/issues/1842))
```

Comment:
When I first read this sentence, I was under the false impression that _all_ `Bit` properties would be checked with `:exhaust`. I think this should mention "when running `:check-docstrings`" upfront, as that is an important caveat. Something like:

```suggestion
* When running the `:check-docstrings` command, `Bit` properties (e.g. `property p = True`) will be checked with `:exhaust`,
  unless their docstrings contain code blocks understood by `:check-docstrings`.
  ([#1842](https://github.com/GaloisInc/cryptol/issues/1842))
```

Reply:
aee7bfbc